rizza/helpers/config.py

```diff
@@ -92,6 +92,25 @@ def _load_genetics(self):
                 'BadValueError': -500,
                 'TypeError': -200
             }
+        if not self.RIZZA['GENETICS'].get('NAILGUN FIELDS TYPE MAPPING'):
+            self.RIZZA['GENETICS']['NAILGUN FIELDS TYPE MAPPING'] = {
+                'BooleanField': 'gen_boolean',
+                'DateField': 'gen_date',
+                'DateTimeField': 'gen_datetime',
+                'DictField': 'dict',
+                'EmailField': 'gen_email',
+                'FloatField': 'float',
+                'IntegerField': 'gen_integer',
+                'IPAddressField': 'gen_ipaddr',
+                'ListField': 'list',
+                'MACAddressField': 'gen_mac',
+                'NetmaskField': 'gen_netmask',
+                'StringField': 'gen_string',
+                'URLField': 'gen_url',
```

Comment:
mapping fields like this may stepping over the line that is providing too much help to Rizza instead of letting Rizza figure things out with experimentation

Reply:
I disagree, because what we want to achieve in first step is fast genetic programming, is always easier to make it more complex without a hints. 

A) Now we are talking about 15 different types, 2 are using other entities, 2 are more complex types lst and dict.
And now they are exactly match.  So complexity is only in choosing the right fields and arguments, not matching the types instead. 

B) In your previous solution you used 37 types, 
`list(entity_tester.EntityTester.pull_input_methods()` without float list and dict which are actually used in the nailgun. 
so each field and attribute has to firstly guess the type which is complexity of number_of_fields^field_types + number_of_args^field.  And we didn't even start to match the fields and arguments required for the entities.

A broader search of space in option  B can found results in BZ creation, but it would take forever.  You might run it separately, but anyway you firstly need a way to look at the answer and say okay this is fine/wrong. Doing it manually might be overkill.